hack/tools/Makefile

```diff
@@ -28,7 +28,7 @@ IMGPKG_VERSION=v0.35.0
 KUBECTL_VERSION=v1.26.0
 KIND_VERSION=v0.17.0
 GINKGO_VERSION=v2.9.2
-COSIGN_VERSION=2.0.0
+COSIGN_VERSION=2.0.2
```

Comment:
In go.mod file cosign version is v2.0.3 `github.com/sigstore/cosign/v2 v2.0.3-0.20230519173114-f21081a18209`
Here we are referring 2.0.2 is it correct or we need to use 2.0.3 here also?

Reply:
cosign v2.0.3 is not release yet. We need to use cosign `main`  latest commit as dependency to have the rekor module v1.1.1 included in the cosign library CLI use.

Reply:
Update to use the cosign latest main commit